### ISSUE TITLE
https://fedoraproject.org/  raise TldDomainNotFound

### ISSUE THREAD

**Issue by Weijianlin** [barseghyanartur/tld#49]
tld.exceptions.TldDomainNotFound: Domain www.fedoraproject.org didn't match any existing TLD name!

https://platform.sh/blog/speed-confidence-and-simplicity-what-our-34m-raise-means-for-you

tld.exceptions.TldDomainNotFound: Domain platform.sh didn't match any existing TLD name!

**Comment by barseghyanartur** [comment]
@Weijianlin:

This is really weird. A lot of things have been changed recently. I think you have discovered an interesting bug.

```python
from tld import get_tld
```

This would raise an exception:

```python
get_tld('https://www.fedoraproject.org/')
```

And this would not:

```python
get_tld('https://www.fedoraprojec1.org/')
```

**Comment by Yomguithereal** [comment]
I can take a look at it yes.

**Comment by barseghyanartur** [comment]
Fix will be released today.

**Comment by barseghyanartur** [comment]
Fixed in 0.9.1.
